src/main/groovy/com/morpheusdata/xen/XenserverProvisionProvider.groovy

```diff
@@ -1808,8 +1809,8 @@ class XenserverProvisionProvider extends AbstractProvisionProvider implements Wo
 
 	def buildNetworkInterface(server, networkResults, newNetwork, newIndex, index) {
```

Comment:
having index and new index arguments here is confusing. Only one index is needed, so remove the last argument `index` that isn't used.

Reply:
unused index param removed as suggested